ResearchKit/Common/ORKStepViewController.h

```diff
@@ -357,6 +358,8 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic, weak, readonly, nullable) ORKTaskViewController *taskViewController;
 
+@property (nonatomic, weak, nullable) ORKNavigationContainerView *navigationContainerView;
```

Comment:
Would it follow ResearchKit idioms better to declare this in ORKStepViewController\_Internal.h?

Reply:
Good suggestion, done.